doc/developers/contributing.rst

```diff
@@ -846,6 +846,10 @@ Finally, follow the formatting rules below to make it consistently good:
 * When editing reStructuredText (``.rst``) files, try to keep line length under
   80 characters when possible (exceptions include links and tables).
 
+* Both single and double backticks surrounding text will render as inline
+  literal ('code', e.g., `list`), due to specific configurations set in
+  scikit-learn.
```

Comment:
Shall we recommend using single quotes by default as a result?

Reply:
I think you mean single backtick?